app/src/main/java/com/sma2/sma2/ExerciseFragments/Ex_tapping.java

```diff
@@ -68,12 +68,12 @@ public void open_exercise(){
                 startActivity(intent_ex1);
                 break;
             case 2:
-                intent_ex1 =new Intent(this, Tapping2.class);
+                intent_ex1 =new Intent(this, Tapping1.class);
                 intent_ex1.putExtra("EXERCISE","tapping2");
                 startActivity(intent_ex1);
                 break;
             case 3:
-                intent_ex1 =new Intent(this, Sliding.class);
+                intent_ex1 =new Intent(this, Tapping1.class);
                 intent_ex1.putExtra("EXERCISE","sliding");
                 startActivity(intent_ex1);
                 break;
```

Comment:
Why in bot cases you have Tapping1. I think one is Tapping2 and the other one is Sliding
Please fix it